CHANGELOG-1.0.md

```diff
@@ -0,0 +1,51 @@
+# Changes since v1.0.1
+
+## New Features
+
+- Users can now provide a secret name and namespace during provision by passing the correct storage class parameters: "provisioner-secret-name" and "provisioner-secret-namespace" ([#281](https://github.com/kubernetes-csi/external-provisioner/pull/281), [@oleksiys](https://github.com/oleksiys))
```

Comment:
We should be specific here `csi.storage.k8s.io/provisioner-secret-name` and `csi.storage.k8s.io/provisioner-secret-namespace`

Reply:
fixed